contrib/basicauth/nginx/registry.conf

```diff
@@ -0,0 +1,17 @@
+# Docker registry proxy for api version 2
+
+upstream docker-registry-v2 {
+  server registryv2:5000;
+}
+
+
+server {
+  listen 5441;
+  server_name localhost;
+  ssl on;
```

Comment:
Here again, this is based on @dmcgowan material.  I'll differ to him for a response.